actions/hadoop-upgrade

```diff
@@ -0,0 +1,253 @@
+#!/bin/bash
+action-set charmdir.before=${CHARM_DIR}
```

Comment:
I think we need a set -e to make sure that if anything goes wrong we exit. I run into a case where the resource was not downloaded and I did not know about it.